CONTRIBUTING.md

```diff
@@ -1,181 +1,163 @@
+If you are working on a previously posted issue, please comment on that issue and let us know you are working on it. That prevents duplicating efforts or having your changes rejected because someone else is working on it.
 
-If you’re interested in adding your new feature to the core Quire default theme, keep in mind some of the questions the core team and the community will be asking:
+#### Post a New Issue
 
-- Does this feature add something new that would be useful to a broad section of Quire’s user base?
-- Does this feature positively effect Quire’s primary tenants of producing publications optimized for discoverability, longevity, beauty and accessibility?
-- Does this feature integrate seamlessly in Quire’s current architecture and will it be reasonable to maintain?
+If your changes are small or straightforward then there is no need to create a new issue. Just dive in and get to work. However, if you are proposing or attempting a large/complex change please post a new issue [on our issue tracker](https://github.com/thegetty/quire-docs/issues/) before proceeding. This helps avoid changes being rejected before you've done too much work. Be sure to briefly describe your proposed solution and say whether you may need help with any aspect of it. This approach has several advantages:
 
-Even if you plan on developing your custom theme/feature independently and offering it to the Quire community as an add on, we highly recommend posting about it on the Quire issue tracker. This lets people know what you’re working on, gives them a chance to offer help, and can bring you valuable feedback and exposure for your work.
-
-[View the current roadmap](https://quire.getty.edu/about/roadmap)<br />
-[Read the Quire Technical Style Guide](https://github.com/thegetty/quire/wiki/Quire-Technical-Style-Guide)<br />
-[Post your feature idea](https://github.com/thegetty/quire/issues/new)
-
-Quire is very extensible. We’re eager to see more development work come from you as you create and customize your own publications in ways that can then be shared back. Quire is in the early stages of defining how this kind of contribution will work, please don’t hesitate to [email us](mailto:quire@getty.edu) with your questions or ideas.
+- Lets people know you’re working on it
+- Gives the core team and the community a chance to give feedback before you do any work
+- Helps to ensure your contribution will be accepted and successfully merged in
 
-### Write an Article
+## Start Work on Contribution
 
-We’re always interested in adding articles about specific aspects of working in Quire to our Learn knowledge base. Maybe you have some tips on modifying shortcodes or styling is CSS in the custom.css. Chances are the community would love to read about it.
+Now that you've identified your contribution, your next step will be to use GitHub and Atom to make the desired changes.
```

Comment:
Do we want to say "text editor" instead of Atom specifically? or "Atom or another text editor"?